Source/WebCore/platform/SimpleCaretAnimator.cpp

```diff
@@ -39,17 +39,23 @@ void SimpleCaretAnimator::updateAnimationProperties(ReducedResolutionSeconds cur
 {
     auto caretBlinkInterval = RenderTheme::singleton().caretBlinkInterval();
 
+    setBlinkingSuspended(!caretBlinkInterval);
+
     // Ensure the caret is always visible when blinking is suspended.
     if (isBlinkingSuspended() && m_presentationProperties.blinkState == PresentationProperties::BlinkState::On) {
-        m_blinkTimer.startOneShot(caretBlinkInterval);
+        m_blinkTimer.startOneShot(caretBlinkInterval.value_or(0_ms));
         return;
     }
 
+    // If blinking is disabled, set isBlinkingSuspended() would have made the
+    // previuos check return early and at this point there must be an interval.
```

Comment:
```suggestion
    // previous check return early and at this point there must be an interval.
```

Reply:
Thanks, I will fix the typo before landing 👌🏼